petab/v1/problem.py

```diff
@@ -50,7 +52,9 @@ class Problem:
     - observables table
     - mapping table
```

Comment:
No mapping in v1? Fine for this PR

Reply:
No mapping in v1 in principle, but it was added there long ago for initial v2 support before we decided to have a separate `v2.Problem`. We should keep it there until petab.v2 has converged.